lambda-on-binaris/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "lambda-on-binaris",
+  "version": "1.0.0",
+  "private": true,
```

Comment:
maybe make it a real wrapper, and the example (=handler) can be its test
(not really sure if it's better, since i don't think this is really useful)